### DISCUSSION TITLE
[2주차] 디자인 회의 및 요청사항

### DISCUSSION THREAD

**Discussion by ohtmm** [jeiltodo/client#6] [General]
@Dragonite-Lee 

준영님,   중요한 순서대로 썼어요! 

---
### api 명세서 업데이트

1. 유저 - 그룹
- api: `GET /group/goals/todos`
- 내용: 진행률에 멤버별 기여도 관련 데이터 추가

<img width="301" alt="스크린샷 2024-07-26 오후 4 24 08" src="https://github.com/user-attachments/assets/f3657031-1689-44e8-98e7-f9f3e81ad361">

- api: `GET /groups/{groupId}`
- 내용: 그룹 구성원별 데이터 업데이트
( createUser 를 고려안하고 그룹장 여부를 isLeader로 썼는데, 통일 부탁드려요!)

<img width="301" alt="스크린샷 2024-07-26 오후 4 39 30" src="https://github.com/user-attachments/assets/c630bedc-f0e6-48a8-a830-2eacd405e144">

---
### pull reqeust
eslint 에러 해결 관련 ,풀리퀘 생성했으니 #3  
확인 후에 동의하면 develop으로 머지해주세요!
( + commit 리스트가 이상합니다만,, )

---

### 디자인 회의
1.  **어드민 - 그룹 목표** 
다음주 월요일 중으로,  관리자 서비스에서 아래 2가지를 보다 명확히 구분 필요
  1) 그룹 관리 - 그룹 목표  
    : 단순히 목표 이름/ 진행상황 조회  ( 클릭시, 그룹 게시글 관리 - 목표 상세로 이동)
  2) 게시물 관리 - 그룹 게시글 관리 - 그룹 목표 
    : 목표 정렬/삭제 기능 

2. **준영 담당 드롭다운** 
- 모달 드롭다운 / 테이블 드롭다운은 
UI 의도가 다르기 때문에 padding, rounded을 통일시키기가 어렵다고 하셨어요! 
그래서 따로 작업해야 할 것 같습니다. 

3. **유저 서비스 일정**
- 다음주 월요일부터 컴포넌트는 작업 가능

4. **로고 및 타이틀 변경 논의**
- 제일 투두보다 나은 타이틀 있을지, 같이 고민했어요
- 제이 ( mbti의 J) 투두  어떤가 이야기했는데, 좀더 나은 타이틀 나올 수도 있으니 더 생각해보기로!

**Comment by aajy** [comment]
hex코드 20개 컬러 리스트는 여기에 우선 남겨둘게요!

#B07100
#FF284F
#FF7020
#FFAD29
#FFEB3E
#FF769C
#AE3BF6
#BBC4DE
#6E7089
#6A3B0C
#27F15D
#17BF43
#0D6000
#044109
#870E02
#3BE0F6
#356EFF
#00639D
#231476
#000132

**Comment by aajy** [comment]
**추가로 공유드릴 사항**

> [wiki - 깃 커밋 전략](https://github.com/jeiltodo/client/wiki/pac.-%EA%B9%83-%EC%BB%A4%EB%B0%8B-%EC%A0%84%EB%9E%B5#footer%EC%84%A0%ED%83%9D%EC%82%AC%ED%95%AD)에 footer 부분 업데이트

- 목적 
  : 커밋 메세지 작성 시 이슈번호를 함께 작성해
깃 이슈 페이지에서 해당 이슈와 관련된 **커밋 히스토리**를 확인할 수 있습니다.
